frontend/app/helpers/format-duration-short.js

```diff
@@ -0,0 +1,10 @@
+import Helper from '@ember/component/helper'
+import { padStartTpl } from 'ember-pad/utils/pad'
```

Comment:
This is not necessary anymore since you can use `String.padStart` now..

Reply:
:+1: